generators/client/templates/angular/src/main/webapp/app/account/settings/_settings.component.html

```diff
@@ -54,19 +54,19 @@ <h2 jhi-translate="settings.title" translate-values="{username: '{{settingsAccou
                     <input type="email" class="form-control" id="email" name="email" placeholder="{{'global.form.email.placeholder' | translate}}"
                            [(ngModel)]="settingsAccount.email" minlength=5 maxlength=100 #emailInput="ngModel" required>
                     <div *ngIf="emailInput.dirty && emailInput.invalid">
-                        <p class="help-block"
+                        <p class="form-text text-muted"
                            *ngIf="emailInput.errors.required" jhi-translate="global.messages.validate.email.required">
                             Your e-mail is required.
                         </p>
-                        <p class="help-block"
+                        <p class="form-text text-muted"
                            *ngIf="emailInput.errors.email" jhi-translate="global.messages.validate.email.invalid">
                             Your e-mail is invalid.
                         </p>
-                        <p class="help-block"
+                        <p class="form-text text-muted"
                            *ngIf="emailInput.errors.minlength" jhi-translate="global.messages.validate.email.minlength">
                             Your e-mail is required to be at least 5 characters.
                         </p>
-                        <p class="help-block"
+                        <p class="form-text text-muted"
```

Comment:
I think it s much better to use `form-text text-danger` here. And it would be nice if we place it in a `<small> element inside the `<p>` but thats optional